src/commcare_cloud/ansible/roles/formplayer/vars/main.yml

```diff
@@ -1,5 +1,6 @@
 formplayer_build_dir: "{{ www_home }}/formplayer_build"
 formplayer_data_dir: "{{ encrypted_root }}/formplayer"
+formplayer_logging_dir: "{{ formplayer_data_dir }}/log"
```

Comment:
I made https://github.com/dimagi/commcare-cloud/compare/dmr/efs+kev/formplayer-requests-logging-config to combine this branch with https://github.com/dimagi/commcare-cloud/pull/3917. The one extra commit I added to make them play nicely was https://github.com/dimagi/commcare-cloud/commit/8b98e7b9665b0562877b8087862d693277d14345, which changes this line to
```
formplayer_logging_dir: "{{ encrypted_root }}/formplayer/log"
```
since the data dir points to the shared network drive and we want logging on /opt/data

Reply:
Awesome, thanks @dannyroberts. I added this [commit](https://github.com/dimagi/commcare-cloud/pull/3914/commits/45bf4cbe1603dd33889f84181590fc9781c95df8) which deals with the flag that we talked about. Do you want me to commit it to the new branch and make a PR from that?